jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/FloydWarshallShortestPaths.java

```diff
@@ -298,16 +310,6 @@ private void lazyCalculateMatrix()
                 }
```

Comment:
@d-michail do we really need the Double comparators instead the of the more efficient literal comparison?